vbox_create.sh

```diff
@@ -166,7 +155,7 @@ function create_cluster_VMs {
   # Gather VirtualBox networks in use by bootstrap VM
   oifs="$IFS"
   IFS=$'\n'
-  bootstrap_interfaces=($($VBM showvminfo ${CLUSTER_PREFIX}bcpc-bootstrap \
+  bootstrap_interfaces=($($VBM showvminfo ${BACH_CLUSTER_PREFIX}bcpc-bootstrap \
```

Comment:
How shall this dependency on the environment be documented? I'd rather see the discivery done or some other way involved to let one know this shell script depends on this obscure variable.

Reply:
vbox_create.sh used to discover prefix by introspecting an environment json file, now it is a variable exported manually and reexported again by tests/automated_install.sh.  Either method requires that a users has knowledge of how to force namespacing.  The new method is now documented in README.md